### PR TITLE
feat: token bridge cpi event 

### PR DESCRIPTION
**Pull request by md0x**
Changes proposed in this PR:
- Emit event from `bridge_tokens_to_hub_pool` function

### CODE REVIEW

File: programs/svm-spoke/src/lib.rs [hunk 1]
```diff
@@ -194,11 +194,18 @@ pub mod svm_spoke {
...
         )
     }
-    pub fn bridge_tokens_to_hub_pool(
-        ctx: Context<BridgeTokensToHubPool>,
+    pub fn bridge_tokens_to_hub_pool<'info>(
+        ctx: Context<'_, '_, '_, 'info, BridgeTokensToHubPool<'info>>,
         amount: u64,
     ) -> Result<()> {
-        ctx.accounts.bridge_tokens_to_hub_pool(amount, &ctx.bumps)?;
+        let context = Context {
```
Comment (md0x) [inline programs/svm-spoke/src/lib.rs:201]: Workaround to be able to pass the context required by `emit_cpi!`

Comment (Reinis-FRP) [reply programs/svm-spoke/src/lib.rs:201]: I think it would be more consistent with other instructions to use free functions and pass ctx as parameter.

Comment (Reinis-FRP) [reply programs/svm-spoke/src/lib.rs:201]: that should also hopefully avoid account cloning below

File: programs/svm-spoke/src/lib.rs [hunk 2]
```diff
@@ -194,11 +194,11 @@ pub mod svm_spoke {
             proof,
         )
     }
-    pub fn bridge_tokens_to_hub_pool(
+    pub fn bridge_tokens_to_hub_pool<'info>(
```
Comment (Reinis-FRP) [inline programs/svm-spoke/src/lib.rs:197]: do we strictly need this info lifetime here? its not being passed in `Context<BridgeTokensToHubPool>` below

**Review by Reinis-FRP — APPROVED** [review]

**Pull request merged**